third_party/nvidia/backend/driver.c

```diff
@@ -306,6 +306,7 @@ static PyObject *fill1DTMADescriptor(PyObject *self, PyObject *args) {
     break;
   default:
     PyErr_SetString(PyExc_ValueError, "elementSize must be 1, 2, or 4");
+    return NULL;
```

Comment:
Another drive-by. `PyErr_SetString` requires the caller to return `NULL` to propagate the exception into python.